draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3774,32 +3772,90 @@ Input: u, an element of F.
 Output: (x, y), a point on E.
 
 Constants:
-1.  c1 = -B / A
-2.  c2 = -1 / Z
+1.  c1 = Z / h
```

Comment:
```suggestion
1.  c1 = Z / S
```

Reply:
By the way, I have not checked this code carefully for correctness. I'll do that once the sqrt_ratio code is fixed.